### ISSUE TITLE
Request: Current highest bid, when searching for the bin value

### ISSUE THREAD

**Issue by Fanatico1981** [Mardaneus86/futwebapp-tampermonkey#149]
READ BEFORE YOU ADD AN ISSUE: Do not request autobuyer features, because they are considered as cheating. Issues regarding autobuyers will be closed immediately.

### Expected behavior:
Beside the BIN calculation, I would like to see the current highest bid, if there is one, so I can adjust the price easier to my expectations. 

### Current behavior:
Only the bin will be calculated

**Comment by Mardaneus86** [comment]
There is no way to filter auctions on the market with actual bids, so I doubt this can be built.

How would you do it manually?

**Comment by Fanatico1981** [comment]
What I do atm.

I click on the button do start your bin calculation.
In the meanwhile I make a manual price check, and check the current biddings of the players on the first site.
After a few seconds your calculated bin price is shown in the browser notification.

**Comment by Mardaneus86** [comment]
Oh, that wouldn't be a perfect implementation if you only look at the first page. So I'm afraid it would cause a lot of confusion on what is the highest bid.

**Comment by Fanatico1981** [comment]
Yeah, normally you should check every auction, but because I do it manually I only check the first page.
But while you check the BIN value, you also see the currentBid, and so you only have to choose the max value of it.

**Comment by Fanatico1981** [comment]
I expect all currentBids to check, not only the one on the first page.
You check the auction info anyway, so you can read as well the currentBid value.
Or not?

**Comment by Mardaneus86** [comment]
Yes, but for min bin we can make use of the filters to find the lowest. For the maximum bids we cannot do that. We simply cannot loop over all pages to find the highest bid, it would cause too many requests. And by filtering you cannot determine whether it is an actual bid or a start price.

**Comment by Mardaneus86** [comment]
Hehe, yes, I agree. But it's EA's decision to it that way. I want to prevent giving you a ban risk for using this tool at all costs.

Thanks for understanding!
